src/util.ts

```diff
@@ -24,3 +26,23 @@ export const getProjectIdFromSentryDsn = (dsn: string) => {
   const search = splitUrlIntoParts(dsn)[5];
   return search.replace('/', '');
 };
+
+const isError = (exception: string | Error): exception is Error => {
+  return (exception as Error).message !== undefined;
+};
+
+/**
+ * Get the message and name properties from the original exception
+ * @param {EventHint} hint
+ */
+export const getOriginalExceptionProperties = (hint: EventHint) => {
+  let name = '';
+  let message = '';
+  if (isError(hint.originalException)) {
+    const originalException = hint.originalException as Error;
+    name = originalException.name;
+    message = originalException.message;
```

Comment:
If you make the othe change I suggested, you could do something like:
```
const {name, message} = originalException;
return {name, message} ;
```

Reply:
this is done.